src/benchmarks/micro/libraries/System.Numerics.Vectors/Perf_VectorConvert.cs

```diff
@@ -0,0 +1,232 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.Numerics.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.SIMD)]
```

Comment:
Is https://github.com/dotnet/runtime/issues/9766 going to need a change in JIT? If so, could you please change the category to `Runtime`?

```suggestion
    [BenchmarkCategory(Categories.Runtime, Categories.SIMD)]
```

Reply:
The goal would actually be to remove the code for these methods from the JIT and build them out using HWIntrinsics only.  Not sure if that changes the category choice or not...

Reply:
In such case `Libraries` is better. Thanks for explanation!